methods.py

```diff
@@ -1,33 +1,38 @@
-from selenium.webdriver.common.by import By
 from selenium.webdriver.support import expected_conditions as EC
```

Comment:
Maybe `webdriver_wrapper`?

Reply:
It is for file and class name.

Reply:
Do you mean it will be inherited by other classes ?